tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/AbstractTychoMapping.java

```diff
@@ -198,24 +198,27 @@ protected Parent findParent(Path projectRoot, Map<String, ?> projectOptions) thr
         Model parentModel = parentPom.getReader().read(parentPom.getPomFile(), options);
         Parent parentReference = new Parent();
         String groupId = parentModel.getGroupId();
-        if (groupId == null) {
+        Parent grandParent = parentModel.getParent();
```

Comment:
Should we just get the grandparent, or keep going up until we find a parent that has a groupId?

Reply:
I think a parent reference must always contain a GAV you can only leave it out for the item itself.